src/main/scala/zio/prelude/NonEmptyList.scala

```diff
@@ -435,16 +435,6 @@ object NonEmptyList extends LowPriorityNonEmptyListImplicits {
   implicit def NonEmptyListAssociative[A]: Associative[NonEmptyList[A]] =
     Associative.make(_ ++ _)
 
-  /**
```

Comment:
I would keep `NonEmptyListIdentityFlatten` and `ZPureIdentityFlatten` in their companion objects.

@sideeffffect WDYT?

Reply:
Yes, we should put instances in the companion object whenever possible.

Reply:
Yes, we should put instances in the companion object whenever possible.

Reply:
Done!